backend/src/utils/proxy.ts

```diff
@@ -27,12 +27,13 @@ const notFoundError = (kind: string, name: string, e?: any, overrideMessage?: st
 };
 
 export const proxyService =
-  <K extends K8sResourceCommon>(
-    model: { apiGroup: string; apiVersion: string; plural: string; kind: string },
+  <K extends K8sResourceCommon = never>(
```

Comment:
ha, I never considered this as a way to make a generic optional. clever.

Reply:
Yes, indeed, I thought the same hahahha